src/odemis/acq/test/move_test.py

```diff
@@ -939,6 +939,40 @@ def test_get_progress_lin_rot(self):
 
         assert progress_0 < progress_1 < progress_2 < progress_3 < progress_end
 
+class TestMoveUtil(unittest.TestCase):
+    """
+    This class is to test movement utilities in the move module
+    """
+
+    def test_has_axis_move(self):
+        """
+        Test has_axis_move function behaves as expected
+        """
+        # positive tests
+        start = {'x': 0.023, 'y': 0.032, 'z': 0.01, "rx": 0, "rz": 0}
+        end = {'x': 0.082, 'y': 0.01, 'z': 0.028, "rx": 0.12213888553625313  , "rz":  5.06145}
+        
+        self.assertTrue(has_axis_move(start, end, 'x'))
+        self.assertTrue(has_axis_move(start, end, 'y'))
+        self.assertTrue(has_axis_move(start, end, 'z'))
+        self.assertTrue(has_axis_move(start, end, 'rx'))
+        self.assertTrue(has_axis_move(start, end, 'rz'))
+
+        # negative tests
+        start = {'x': 0.023, 'y': 0.032, 'z': 0.01, "rx": 0, "rz": 0}
+        end = {'x': 0.023, 'y': 0.032, 'z': 0.01, "rx": 0, "rz": 0}
```

Comment:
If the values are identical, it's a little too easy. Please add some epsilon to at least a couple of axes!